schema/course.model.schema

```diff
@@ -50,6 +50,17 @@
       "translatable": true,
       "help": "This will be added to the course's manifest file metadata. It is not (currently) displayed anywhere within the course."
     },
+    "version": {
+      "type": "string",
+      "required": true,
+      "default": "1.0.0",
+      "inputType": "Text",
+      "validators": [
+        "required"
+      ],
+      "translatable": true,
```

Comment:
It's not translatable. This is for fields that would need to be translated between written languages.